src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1988,6 +1996,13 @@ image::upload-catalog-objects.png[align=left]
 
 WARNING: In case of uploading an archive, all the objects in the zip must have the same Kind and Content Type.
 
+==== Create a zip archive with Catalog objects
+When the user creates a zip archive for uploading objects to Catalog, the hidden files should be excluded from the archive.
+
```

Comment:
When a user creates an archive in order to upload a set of Catalog objects, hidden files should be excluded from this archive.